core/evm.go

```diff
@@ -51,10 +50,10 @@ type ChainContext interface {
 
 // NewEVMBlockContext creates a new context for use in the EVM.
 func NewEVMBlockContext(header *types.Header, chain ChainContext, author *common.Address) vm.BlockContext {
-	var predicateResults *results.PredicateResults
+	var predicateResults *predicate.PredicateResults
```

Comment:
Could we change the type from `PredicateResults` to `Results` so it doesn't stutter?